internal/pkg/cli/run_local.go

```diff
@@ -20,14 +20,17 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/arn"
+	"github.com/aws/aws-sdk-go/aws/request"
 	"github.com/aws/aws-sdk-go/aws/session"
 	sdkecs "github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/aws/aws-sdk-go/service/rds"
```

Comment:
We are not wrapping the dependency - I am a little 🤏🏼  wary of this, but I don't see a strong argument against this usage either. I can be see why it can be okay okay in this specific use case:  a) it is extremely unlikely that we will switch the dependency, i.e. from aws-sdk-go/service/rds to another rds package (which probably doesn't exist) b) the aws-sdk-go interfaces can be _fairly_ stable c) the host finding logics are unique to run local. Are these considerations align with yours too?

Reply:
yeah! i think in general, my approach would be based on the amount of usage in other places of our code. if there's only a single use case, like there is today, then adding an abstraction layer for that single use case is probably prematurely-generalizing and we are creating more work than benefit. 

if we end up using RDS in [3](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)) places, then i'd say making an abstraction is worth it.